bot/cogs/utils.py

```diff
@@ -257,6 +257,24 @@ def check(m: Message) -> bool:
         embed.description = best_match
         await ctx.send(embed=embed)
 
+    @command(aliases=("poll",))
+    @with_role(*MODERATION_ROLES)
+    async def vote(self, ctx: Context, title: str, *options: str) -> None:
+        """
+        Build a quick voting poll with matching reactions with the provided options.
+
+        A maximum of 20 options can be provided, as Discord supports a max of 20
+        reactions on a single message.
+        """
+        if len(options) > 20:
```

Comment:
Also worth checking when there are 0 options. It won't explode or anything but it's probably a user error if there are 0 options.

Reply:
Good point, didn't think of that.

Reply:
I've instead checked if less than 2 options have been provided, as there's no point on voting for a single item.